### PR TITLE
Update workspace-command.md

### PR DESCRIPTION
**Pull request by AlexandraRamirezMS**
Re: [MicrosoftDocs/azure-devops-docs] Update workspace-command.md (PR #13502)

### CODE REVIEW

**Comment by Jak-MS** [comment]
@steved0x @JKirsch1 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/repos/tfvc/workspace-command.md [hunk 1]
```diff
@@ -114,6 +114,29 @@ If you delete a workspace that contains pending changes, TFVC cancels the pendin
 > [!NOTE]
 > Commands that run manually require the `/noprompt` option to bypass user acknowledgement. Be careful if you use the PowerShell `Start()` method to run commands. The `/noprompt` option can be automatically set in PowerShell.
 
+
+When deleting a workspace you need to provide the `<workspace-owner>` and `<workspace-name>`
```
Comment (JKirsch1) [inline docs/repos/tfvc/workspace-command.md:118]: When you delete a workspace, you need to provide values for the `<workspace-owner>` and `<workspace-name>` parameters.

File: docs/repos/tfvc/workspace-command.md [hunk 2]
```diff
@@ -114,6 +114,29 @@ If you delete a workspace that contains pending changes, TFVC cancels the pendin
...
+
+You can use the `<workspace` command to retrieve those values. For more information, see [workspaces command](https://learn.microsoft.com/en-us/azure/devops/repos/tfvc/workspaces-command?view=azure-devops)
```
Comment (JKirsch1) [inline docs/repos/tfvc/workspace-command.md:120]: You can use the `workspaces` command to retrieve those values. For more information, see [Workspaces command](workspaces-command.md).

File: docs/repos/tfvc/workspace-command.md [hunk 3]
```diff
@@ -114,6 +114,29 @@ If you delete a workspace that contains pending changes, TFVC cancels the pendin
...
+```
+
+To delete the workspace, run the following command:
+
+```
+c:\projects>tf workspace /delete <workspace-name>;<workspace-owner> /collection:<team-project-collection-url>`
+```
+
+
 ### Edit a workspace
```
Comment (JKirsch1) [inline docs/repos/tfvc/workspace-command.md:141]: Thanks for the updates! I'd just recommend making the few minor changes that I've listed in the comments to improve accessibility and correct typos.

**Review by steved0x — APPROVED** [review]

**Comment by steved0x** [comment]
#sign-off

File: docs/repos/tfvc/workspace-command.md (hunk 2 above)
Comment (Jak-MS) [inline docs/repos/tfvc/workspace-command.md:120]:
```suggestion
You can use the `workspace` command to retrieve those values. For more information, see [workspaces command](/azure/devops/repos/tfvc/workspaces-command?view=azure-devops).
```

**Pull request merged**